BUILD.gn

```diff
@@ -937,6 +937,76 @@ if (is_mac) {
       "@executable_path/../Frameworks",
     ]
   }
+
+  if (enable_dsyms) {
```

Comment:
This never appears to be set? Unless I'm missing a thing?

Reply:
it's set in Chromium's GN files.